tasks/main.yml

```diff
@@ -9,5 +9,5 @@
   tags: script
   no_log: true
 - name: Run verification script
-  shell: /root/ceph-version-qemu.sh
+  shell: /root/ceph-version-qemu.sh "{{ '-e' if exclude_no_rbd=='true' else '' }}" "{{ '-d' if script_debug=='true' else '' }}"
```

Comment:
I can't get the extra arguments to work with ansible.

```
$ ansible-playbook -i inventories/hosts.file ops_playbooks/ceph-version-qemu.yml -e script_debug=True  -vvv
```

Does not print the debug output from the bash script:

```
ok: [mycloudnode2.example.org] => {
    "changed": false, 
    "cmd": "/root/ceph-version-qemu.sh \"\" \"\"", 
    "delta": "0:00:02.789636", 
    "end": "2019-03-14 12:36:31.198899", 
    "invocation": {
        "module_args": {
            "_raw_params": "/root/ceph-version-qemu.sh \"\" \"\"", 
            "_uses_shell": true, 
            "chdir": null, 
            "creates": null, 
            "executable": null, 
            "removes": null, 
            "stdin": null, 
            "warn": true
        }
    }, 
    "rc": 0, 
    "start": "2019-03-14 12:36:28.409263", 
    "stderr": "", 
    "stderr_lines": [], 
    "stdout": "UID UID2, 
    "stdout_lines": [
        "UID2", 
        "UID1d"
    ]
}
```

I think that's because the script is asking for true, which apparently is not a boolean. Maybe that could be adjusted? (then also putting in True / False or yes/no and the other yaml variants would work)

Reply:
That's a clear bug that in my testing happened to work since I used true and it was expecting a literal 'true'. Should be fixed now